### PR TITLE
Problem: docs fail to mention bindings

### PR DESCRIPTION
**Pull request by dkliban**
Solution: provide links to rubygems.org and pypi.org

This patch also provides documentation on generating bindings in any language.

re: #4687
https://pulp.plan.io/issues/4687

### CODE REVIEW

File: docs/integration-guide/index.rst [hunk 1]
```diff
@@ -1,4 +1,46 @@
...
+
+``pulpcore_client`` Ruby Gem is available on
+`rubygems.org <https://rubygems.org/gems/pulpcore_client>`_. It is currently published daily and
+with every RC. Each plugin is responsible for publishing it's own client to Rubygems.org. The
+client libraries for plugins should follow the``pulp_<slug>_client`` naming scheme.
+
+Client in language of your choice
+=================================
+
+Both Python and Ruby clients are generated using Pulp's OpenAPI 2.0 schema and `openapi-generator
```
Comment (bmbouter) [inline docs/integration-guide/index.rst]: This first sentence seems off topic. Maybe remove it.

File: docs/integration-guide/index.rst [hunk 2]
```diff
@@ -1,4 +1,46 @@
...
+<https://openapi-generator.tech>`_. A client can be generated using any of the available `generators
```
Comment (bmbouter) [inline docs/integration-guide/index.rst]: Maybe rewrite this sentence to be:

Bindings can be generated in many languages using the available generators.  <-- with the link.

File: docs/integration-guide/index.rst [hunk 3]
```diff
@@ -1,4 +1,46 @@
...
+<https://openapi-generator.tech/docs/generators.html>`_.
+
+The OpenAPI schema for pulpcore and all installed plugins can be downloaded from the ``pulp-api``
```
Comment (bmbouter) [inline docs/integration-guide/index.rst]: This needs some scaffolding for the reader (I think). Maybe above it say, building bindings in language foo is done in two steps:

1. Download the OpenAPI schema from pulpcore (this defines the API of Pulp and all installed plugins on your Pulp server)
2. Build the bindings with ...   <-- I'm not sure what to write here

Comment (bmbouter) [reply docs/integration-guide/index.rst]: I also thought this would have building instructions or maybe a link to them?

Comment (dkliban) [reply docs/integration-guide/index.rst]: That sounds good. I just pushed an update.

File: docs/integration-guide/index.rst (hunk 2 above)
Comment (dkliban) [reply docs/integration-guide/index.rst]: Yep. Pushed an update.

File: docs/integration-guide/index.rst (hunk 1 above)
Comment (dkliban) [reply docs/integration-guide/index.rst]: removed

File: docs/integration-guide/index.rst [hunk 4]
```diff
@@ -1,4 +1,52 @@
...
+
+.. code-block:: bash
+
+    curl -o api.json http://<pulp-hostname>:24817/pulp/api/v3/docs/api.json?plugin=pulp_rpm
+
+**2) Generate a client using openapi-generator.**
+
+The schema can then be used as input to the openapi-generator-cli. The documentation on getting
+started with openapi-generator-cli is available on
+`openapi-generator.tech <https://openapi-generator.tech/#try>`_.
```
Comment (bmbouter) [inline docs/integration-guide/index.rst]: I thought this would use https://github.com/pulp/pulp-swagger-codegen  ?

Comment (dkliban) [reply docs/integration-guide/index.rst]: That only supports Python and Ruby. Each generator takes different configuration parameters. I baked in the config only for Python and Ruby.

Comment (bmbouter) [reply docs/integration-guide/index.rst]: oic. Thank you

**Pull request merged**